api/src/main/resources/metadata/concepts_and_drugs/Concept.xml

```diff
@@ -4877,4 +4877,5 @@
     <concept  concept_id="175393"  retired="0" datatype_id="4"  class_id="5"  is_set="0"  creator="1"  date_created="2024-05-09 13:41:10"  version=""  changed_by="1"  date_changed="2024-05-09 13:41:10" uuid="c07bd21c-8a0d-4812-9887-3308ee7f5e42"/>
     <concept  concept_id="175392"  retired="0" datatype_id="2"  class_id="1"  is_set="0"  creator="1"  date_created="2024-05-09 12:45:52"  version=""  changed_by="1"  date_changed="2024-05-09 12:45:52" uuid="b2e6884d-2354-4d93-ab22-ec9d2b262177"/>
     <concept  concept_id="175391"  retired="0" datatype_id="2"  class_id="1"  is_set="0"  creator="1"  date_created="2024-05-09 12:40:41"  version=""  changed_by="1"  date_changed="2024-05-09 13:48:23" uuid="96370f02-aab1-4ec2-a42e-df73f63f12d4"/>
+    <concept  concept_id="175402"  retired="0" datatype_id="4"  class_id="11"  is_set="0"  creator="1"  date_created="2024-06-04 09:42:43"  version=""  changed_by="1"  date_changed="2024-06-04 09:42:43" uuid="17bda0f8-e29c-48a4-8183-fb2c24891bc9"/>
```

Comment:
@semujju  this concept exists as "Other specified acquired abnormality of congenitally malformed valve". You always need to update your environment before creating any new concept